tests/tests.toml

```diff
@@ -152,6 +152,11 @@ asr = true
 filename = "../integration_tests/expr_10.py"
 asr = true
 
+[[test]]
+filename = "../integration_tests/expr_11.py"
+c = true
+cpp = true
+
```

Comment:
Fine to have this but not necessary. We are already testing the generated code via `integration_tests/CMakeLists.txt`.